### PR TITLE
Update CHANGELOG and CONTRIBUTION files

### PR DESCRIPTION
**Pull request by bastienGranger**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ These instructions will get you a copy of the project up and running on your loc
 
 ### Prerequisites
 
-You need to install on your machine :
+You need to install:
 
-- a recent version of [Node.js](https://nodejs.org/)
+- [Node.js](https://nodejs.org/)
```
[the remark this answers is not in the document]
Comment (bastienGranger) [reply CONTRIBUTING.md]: Not a big deal cause first if you install from the website you install the latest unstable or the latest stable version and both version are recent. If a non recent enough recent version is already installed the a `npm i` will fail with an error message `need node version x`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -60,15 +60,15 @@ All you need to do is :
...
 
 ### Good to know :
 
-- It's convenient to configure your ide in order automatically format your files. Please follow this [Editor Integration documentation](https://prettier.io/docs/en/editors.html).
+- It's convenient to configure your IDE in order automatically format your files. Please follow this [Editor Integration documentation](https://prettier.io/docs/en/editors.html).
 
 - Otherwise, we have a yarn task that formats your <strong>git staged files</strong> for you. You can simply run `yarn pretty-quick`
 
-- We also have an automatic task that runs a <strong>prettier</strong> and <strong>eslint</strong> check on git staged files everytime you attempt to create a new <strong>commit</strong> in order to be sure that things are OK
+- We also have an automatic task that runs a <strong>prettier</strong> and <strong>eslint</strong> check on git staged files everytime you attempt to create a new <strong>commit</strong> in order to be sure that thing is OK
```
[the remark this answers is not in the document]
Comment (bastienGranger) [reply CONTRIBUTING.md]: everything is OK I missed the every

File: README.md [hunk 3]
```diff
@@ -4,16 +4,11 @@ This library contains Dashlane's reusable ui components built with React 🎉
 
 ## Getting Started
 
-These instructions will get you the latest version of the `ui-components` lib installed in your project, so you start integrating ans using the components.
```
[the remark this answers is not in the document]
Comment (bastienGranger) [reply README.md]: IMO it is useless. And I think that we should keep only useful info on this document. The shorter and precise, the better

File: README.md [hunk 4]
```diff
@@ -4,16 +4,11 @@ This library contains Dashlane's reusable ui components built with React 🎉
...
-
-### Compatibility
```
[the remark this answers is not in the document]
Comment (bastienGranger) [reply README.md]: I'll keep it though

File: README.md [hunk 5]
```diff
@@ -42,7 +37,7 @@ const App: React.FC = () => {
           Edit <code>src/App.tsx</code> and save to reload.
         </p>
       </header>
-      <Button title={'Click me'} primary={true} />
+      <Button title={'Click me'} primary />
```
[the remark this answers is not in the document]
Comment (bastienGranger) [reply README.md]: For boolean props on react value is not mandatory. `<Button primary={true} />` is equivalent to `<Button primary />

**Comment by pducolin** [comment]
@bastienGranger this PR looks obsolete, can we close it?

**Comment by bastienGranger** [comment]
Old and not relevent

**Pull request closed without merging**